### DISCUSSION TITLE
[Support] swagger的index.d.ts的错误

### DISCUSSION THREAD

**Discussion by minefujiko9293** [midwayjs/midway#4205] [Q&A]
### Describe the problem（描述问题）

![image](https://github.com/user-attachments/assets/81ca69bd-8717-421f-b517-b2f1416e5190)

SchemaObject.required为string[]? 不是boolean吗

![image](https://github.com/user-attachments/assets/55cf664e-df2f-4aa0-99a7-ebf137cf7bfd)

### Midway Versions（Midway 版本）

3.19.0

**Comment by czy88840616** [comment]
https://swagger.io/docs/specification/v3_0/data-models/data-types/
<img width="1072" alt="image" src="https://github.com/user-attachments/assets/19549614-ee5a-4467-a4b7-d0be065eef84">

**Comment by minefujiko9293** [comment]
![image](https://github.com/user-attachments/assets/b1e9dac9-bdbc-44d6-8997-cf954c2c922e)
@czy88840616 
还是不知道咋写。。。
不然就得单独写个类了

**Reply by czy88840616** [reply]
这里有一些自定义对象的例子。 https://github.com/midwayjs/midway/blob/main/packages/swagger/test/parser.test.ts#L1860
